helm/templates/backend/backend.serviceaccount.yaml

```diff
@@ -25,6 +25,10 @@ rules:
 - apiGroups: [""]
   resources: ["pods", "pods/log", "events"]
   verbs: ["get", "list"]
+- apiGroups: [""]
+  resources: ["pods/exec"]
+  verbs: ["get", "create"]
```

Comment:
These lines can be deleted now

Reply:
Upload of files still works in the "old" way and this uses exec to upload the files. Or am I wrong?

Reply:
Oh yes, you're right!